src/igvfd/types/file.py

```diff
@@ -103,6 +103,13 @@ class File(Item):
         Path('submitted_by', include=['@id', 'title']),
     ]
 
+    set_status_up = [
+        'derived_from',
+        'file_format_specifications',
+        'file_set'
```

Comment:
I am not sure about file_set. it is potentially possible to release a file in the archived file-set - and it should not trigger the release of the archived file set?